__tests__/errorReporting.spec.js

```diff
@@ -167,18 +167,21 @@ describe('error reporting', () => {
       // needed to handle the asynchronously rejected promise in this synchronous unit test
       // jest version 21.x.x and up no longer tolerate unhandled rejected promises
       // https://github.com/facebook/jest/issues/6028
-      return promise.catch(console.error);
+      return promise.catch(() => 0);
```

Comment:
Why are we returning `0` instead of `console.error`?

Reply:
It's just a noop. No reason to log this error in our tests